auditlog/data/ir_cron.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data noupdate="1">
+
+        <record id="ir_cron_auditlog_autovacuum" model="ir.cron">
+            <field name='name'>AutoVacuum audit logs</field>
```

Comment:
Your call: this is not so important if it is consistent with existing "misspelling" ;)